hpack/hpack.py

```diff
@@ -23,6 +23,12 @@
 INDEX_NEVER = b'\x10'
 INDEX_INCREMENTAL = b'\x40'
 
+# Precompute 2^i for 1-8 for use in prefix calcs.
+# Zero index is not used but there to save a subtraction
+# as prefix numbers are not zero indexed.
+_PREFIX_BIT_MAX_NUMBERS = [(2 ** i) - 1 for i in range(9)]
+_UNROLLED_INDEX_SHIFT = [(i, (i - 1) * 7) for i in range(1, 20)]
```

Comment:
Ah excellent spot, I was previously using that fixture to unroll all the additions but it turns out accessing a global object and iterating over it is slower than twice as many additions so I scrapped it. I'll remove that.